apps/client/electron/webpack_electron_main.mjs

```diff
@@ -49,11 +49,11 @@ export default ({sentryDsn, appVersion}) => [
     ],
     output: {
       filename: 'index.js',
-      path: path.resolve(__dirname, '..', '..', 'build', 'electron', 'electron'),
+      path: path.resolve(__dirname, '..', '..', '..', 'build', 'electron', 'electron'),
```

Comment:
Same here, we need to scope this to the client eventually. Perhaps `build/app/client/electron, to mirror the source tree. It doesn't have to be in this PR, but must be before we migrate the Manager.